main.tex

```diff
@@ -241,7 +241,7 @@ \subsubsection*{Browser use}
 \end{minted}
 \end{small}
 \caption{\textbf{Basic use, including appending data to formatted bibliography entries.}
-Here, line 3 loads the Altmetric widget code, line 4 loads the library, line 10 imports \texttt{Cite}, and line 11 creates an interface for a bibliography with one entry, with metadata from a DOI. Lines 13-18 render the bibliography, with line 14 setting the output to HTML and lines 15-17 appending an Altmetric widget to the entry. Lines 20-21 show the output on the page. Lines 9 and 22 are to avoid race conditions in DOM access. Line 3 and 22 initialize the Altmetric badge. In the example, \citep{hallmann_more_2017} is used.
+Here, line 3 loads the Altmetric widget code, line 4 loads the library, line 10 imports \texttt{Cite}, and line 11 creates an interface for a bibliography with one entry, with metadata from a DOI. Lines 13-18 render the bibliography, with line 14 setting the output to HTML and lines 15-17 appending an Altmetric widget to the entry. Lines 20-21 show the output on the page. Lines 9 and 22 are to avoid race conditions in DOM access. Lines 3 and 22 initialize the Altmetric badge. In the example, \citep{hallmann_more_2017} is used.
```

Comment:
"Lines 9 and 22 are to avoid race conditions in DOM access."

By the way, is the reference to line 22 here correct? I was just wondering if you meant to refer to the "_altmetric_embed_init()" line, or if you are off by one and were trying to refer to the closing bracket of the `window.onload = async function () {` function.

Line numbers:

![image](https://user-images.githubusercontent.com/77951/61217335-71476b80-a6dd-11e9-9d72-bbd27e196261.png)